src/olympia/abuse/models.py

```diff
@@ -1346,6 +1350,7 @@ def requeue_held_action(self, *, user, notes):
             job = CinderJob.objects.create(
                 target_addon=self.addon,
                 resolvable_in_reviewer_tools=True,
+                job_id=None,
```

Comment:
Is this for readability or do we have to specify none here?

Reply:
it's necessary - the default is still `''` (though the logic for working out what a default value is when undefined is [complicated](https://github.com/django/django/blob/4.2.20/django/db/models/fields/__init__.py#L960))... though maybe I should set a default on the field and disallow any future empty string values...

Reply:
https://github.com/mozilla/addons/issues/15583